### ISSUE TITLE
NMMNH - Family Tides Exhibit

### ISSUE THREAD

**Issue by lin-fred** [ArctosDB/data-migration#257]
Information:
Loan for the molluscs: 2019.001.Inv
Project Name: Family Tides: A Mollusc Family Tree Exhibit

Process

1. Download the Condition Pictures onto the work laptop, and organize them into folders like: Lottidae 4895 etc.
2. Put all of the pictures in their respective folder
3. Add columns in excel sheet for MediaID1, MediaID2 etc, add the media ID to the excel sheet
4. Upload the Condition Pictures to their respective specimens
5. Media type: image
6. Mime type:
7. License: CC BY-NC-SA
8. Created by agent: Lindsey Marie Frederick
9. Description: Condition Report Picture for loan to Family Tides Exhibit
10. Made Date: condition report date
11. Update Part Attributes by using ConditionOstShells excel file
12. the disposition to “on exhibit”
13. Condition will change from unchecked to the condition listed in the excel file
14. Confirm number count
15. If count is higher than in the picture
16. You’ll create a subsample to indicate the number of ones that are on exhibit (please check with teresa to be sure this is the way its need to be done)
17. Remarks put the notes and other info, like this: Condition checked by LF 11/15/2018 Number written on bottom is almost unreadable
18. Link the specimens to the Loan using the part

- http://handbook.arctosdb.org/how_to/How-to-Create-a-New-Loan.html#adding-specimens-to-the-loan

**Comment by Jegelewicz** [comment]
@lin-fred I have been creating barcodes for spaces in the main museum so we can track stuff on exhibit using object tracking. I don't know which hall the exhibit is in, but we can set it up if it isn't already AND we can create a barcode for the case as well to really define WHERE they are.

**Comment by dustymc** [comment]
An app that can scan those barcodes and pull data from Arctos' API would be pretty cool....

**Comment by lin-fred** [comment]
@Jegelewicz I also don't know what hall it would be considered, it's sort of inbetween - so once we are able to be back I can try and figure that out. Definitely should be the last step of this "project" though

**Comment by Jegelewicz** [comment]
> An app that can scan those barcodes and pull data from Arctos' API would be pretty cool....

I wish - for now our containers only consist of halls (and probably with this cases). Barcodes for individual specimens is a whole other ball game! We can't even get the iPad that's supposed to go with this stuff installed - baby steps....

Oh and never mind that no one has been in the museum for months now.....

**Comment by Jegelewicz** [comment]
@lin-fred we can still create a barcode for the case (or maybe one for each case - however you want to do it) and put the stuff in the cases, then put the case in the building until we get a more refined location.

**Comment by lin-fred** [comment]
Another quick question, Paula is uploading some media, and I thought we used to be able to create a thumbnail during the creation of the media, but that doesn't seem to happen anymore?

Was that lost in the move to postgres? @dustymc

**Comment by dustymc** [comment]
https://github.com/ArctosDB/documentation-wiki/issues/144 - it'll autogenerate a thumb

**Comment by lin-fred** [comment]
oooh awesome!

**Comment by PaulaBarteau** [comment]
When adding parts to the loan, if we encounter a lot with specimens on exhibit and not on exhibit, should we create a sub-sample to split them up? Also, many of our shells need to be counted, so we may retroactively need to split them into subsamples on exhibit and in the collection after we can get an accurate count. How should we handle this?

**Comment by Jegelewicz** [comment]
I suggest creating separate parts, but you could use subsampling if you want. It's really up to @lin-fred 

I just associate subsamples with stuff that ends up never coming back (like tissues for destructive sampling. One thing we didn't talk about is how the shell that was exhibited would be marked as "the one" out of the lot. Would anyone care or will it just be put back in with the rest?

**Comment by PaulaBarteau** [comment]
From Teresa, putting it here so we can find it later
"...To make things simpler, you could make a list of any partless specimens
that belong in the loan and make a small part bulkload file for them
instead of adding them individually. As far as the unknown count, you do
know how many are in exhibit, so I would just create a part with lot count
of 1. If it turns out there are more, they should be added as separate
parts to show that only 1 was on exhibit. I wouldn't use subsample for
this, you just end up with two parts one with lot count 1 (which will be
associated with the loan) and one with lot count of whatever the count is
minus 1"

**Comment by lin-fred** [comment]
> Would anyone care or will it just be put back in with the rest?

These are the nightmares that keep me up at night

There could be many ways to indicate that _one_, sometimes out of two, sometimes out of 100 that look identical, was the one put on exhibit. Would it matter? Mostly no, except for when that one gets damaged, lost or something else that we want to make sure that we always use that one instead of the other 99 for future exhibitions. I'd separate out the one by putting it in a vial but still with the others..or something along those lines. So I care, and will probably separate them out from their lots once back in the collection.

When it comes back, is it better for it to have been its own part, or under the original lot number?
I think after this discussion, I would also like to go with creating another part instead of using a subsample.

So if there is a count of 100, edit the original part to 99 and add in another part with lot count 1, and add that one to the exhibit/loan, just like what Teresa said.

Thanks all!

**Comment by Jegelewicz** [comment]
> When it comes back, is it better for it to have been its own part, or under the original lot number?
> I think after this discussion, I would also like to go with creating another part instead of using a subsample.

The lot number applies to ALL the parts (at least that is how I would do this) and so would be an "original identifier".

Separating "the one" into its own BARCODED container, will work well! You wouldn't need to mark up the specimen, just add the barcode to the vial, then put the vial in the drawer and you are good to go!

**Comment by Jegelewicz** [comment]
The alternative, which in this case is probably overkill is to re-catalog "the one" and associate it with the original catalog number (the lot).

I would only recommend that if it was identified as some taxon other than the rest of the lot.

**Comment by campmlc** [comment]
There is no reason why you can't split lots of the same species. We split
lots of the same species and recatalog as "same lot as" relationship when
the individual catalog item needs to be subsampled for genetic material or
be linked as an individual to external repositories such as GenBank or
MorphoSource.

**Comment by Jegelewicz** [comment]
> when the individual catalog item needs to be subsampled for genetic material or be linked as an  individual to external repositories such as GenBank or MorphoSource.

Yes, but just to say this shell with no data was on exhibit once - it seems like overkill.

**Comment by campmlc** [comment]
In either case, assigning the original lot number as an other identifier is
a very good idea.

**Comment by campmlc** [comment]
true, unless there is associated media of the exhibit?

**Comment by Jegelewicz** [comment]
> true, unless there is associated media of the exhibit?

Maybe, but could also associate the media with the exhibit loan (and therefore sorta with the part) also could add the Barcode as a description on the media.

**Comment by lin-fred** [comment]
@PaulaBarteau please not above in the text that I changed the loan number, it was inconsistent with the other in house loan number format
